skaha/src/main/webapp/service.yaml

```diff
@@ -1,14 +1,14 @@
 swagger: '2.0'
 info:
-  version: 0.3
+  version: 0.4
```

Comment:
I guess this should match the version of skaha, 0.10.0?

Reply:
Okay.